includes/indices/class-algolia-searchable-posts-index.php

```diff
@@ -456,8 +456,9 @@ protected function get_items( $page, $batch_size ) {
 	 * @since  1.0.0
 	 *
 	 * @param mixed $item The item to delete.
+	 * @param bool  $wait Wait the for operation to complete.
```

Comment:
This comment needs correction. It should be "Wait for the operation to complete synchronously."